common/common.scss

```diff
@@ -39,6 +39,11 @@ body.static-login {
 
   .login-welcome {
     font-size: var(--font-up-1);
+    .login-welcome__title {
+      p {
```

Comment:
We should generally avoid to style on direct html elements, a class would most likely be better here.

Reply:
Agreed. I used the element because there was no class name present. I just added on in this [core PR](https://github.com/discourse/discourse/pull/22002), and updated this PR to use that class.